apps/studio/components/interfaces/Database/Backups/DatabaseBackupsNav.tsx

```diff
@@ -29,11 +29,8 @@ function DatabaseBackupsNav({ active }: Props) {
       enabled: databaseRestoreToNewProject && cloud_provider !== 'FLY',
       id: 'rtnp',
       label: (
-        <div className="flex items-center gap-1">
-          Restore to new project{' '}
-          <Badge size="small" className="!text-[10px] px-1.5 py-0">
-            Beta
-          </Badge>
+        <div className="flex items-center gap-2">
+          Restore to new project <Badge>Beta</Badge>
```

Comment:
Very tiny nit but BETA uses default badge here
<img width="226" height="72" alt="image" src="https://github.com/user-attachments/assets/b90da5b4-c7d8-4883-adaf-b0b4af004496" />

whereas in other UI we're using warning badge
<img width="281" height="55" alt="image" src="https://github.com/user-attachments/assets/2d6a1ffe-b3c1-4de7-9bdc-c18c566894ee" />

should we align?